addons/l10n_in/models/res_partner.py

```diff
@@ -63,10 +64,10 @@ def _compute_l10n_in_gst_state_warning(self):
             else:
                 partner.l10n_in_gst_state_warning = False
 
-    @api.depends('l10n_in_pan')
+    @api.depends('l10n_in_pan_entity_id')
     def _compute_display_pan_warning(self):
         for partner in self:
-            partner.display_pan_warning = partner.vat and partner.l10n_in_pan and partner.l10n_in_pan != partner.vat[2:12]
+            partner.display_pan_warning = partner.vat and partner.l10n_in_pan_entity_id and partner.l10n_in_pan_entity_id.name != partner.vat[2:12]
```

Comment:
Is there a case where PAN entity != that part of the name and that it makes legal sense?